runtime/gc_trace/TgcCardCleaning.cpp

```diff
@@ -58,7 +58,7 @@ printCardCleaningStats(OMR_VMThread *omrVMThread)
 	U_64 totalTime = 0;
 	UDATA totalCardsCleaned = 0;
 	
-	omrstr_ftime_ex(timestamp, sizeof(timestamp), "%b %d %H:%M:%S %Y", j9time_current_time_millis(), OMRSTR_FTIME_FLAG_LOCAL);
+	omrstr_ftime_ex(timestamp, sizeof(timestamp), "%b %d %H:%M:%S %Y", j9time_hires_clock(), OMRSTR_FTIME_FLAG_LOCAL);
```

Comment:
It is not the same: `j9time_current_time_millis()` returns time in milliseconds since January 1st 1970 UTC but `j9time_hires_clock()` output is read of high-resolution performance counter (ticks).

Reply:
I guess we can just leave this unchanged, and care only about those that calculate time diff?

Reply:
Yes, this is correct. We need to fix only critical ones (like cases where delta time may be calculated). No need to modify places like general day/time reporting.